src/lms-ds-loader/lms_ds_loader/constants.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+

```

Comment:
The Idea of this file is to have a kind of namespaces, right?

Reply:
Yes. I wanted to re-use these constants - and there will be more eventually - in a few places. I like grouping constants by similarity so that it is easy to find them. Hence "namespace-like" use of nested classes.